client/my-sites/checkout/upsell-nudge/professional-email-upsell/index.tsx

```diff
@@ -158,21 +208,16 @@ const ProfessionalEmailUpsell = ( {
 						comment: '%(domain)s is a domain name, like example.com',
 					} ) }
 				</h1>
-				<div className="professional-email-upsell__pricing">
-					<span
-						className={ classNames( 'professional-email-upsell__standard-price', {
-							'is-discounted': true,
-						} ) }
-					>
-						{ formattedPrice }
-					</span>
-
-					<span>
-						<Badge type="success">{ translate( '3 months free' ) }</Badge>
-					</span>
-				</div>
+				<h3 className="professional-email-upsell__small-subtitle">
+					{ translate( 'No setup required. Easy to manage.' ) }
```

Comment:
Given that we want to get this launched ASAP, and it will take a few days for the translations to flow in, it may make sense to check for the translation before we show it.
```suggestion
					{ i18n.hasTranslation( 'No setup required. Easy to manage.' ) ? translate( 'No setup required. Easy to manage.' ) : null }
```

Reply:
Didn't know that function! ❤️